apps/dashboard/src/pages/usecase-select-page.tsx

```diff
@@ -101,6 +101,11 @@ export function UsecaseSelectPage() {
 
   return (
     <>
+      <Helmet>
+        {channelOptions.map((option) => (
+          <link key={option.id} rel="prefetch" href={`/images/auth/${option.image}`} as="image" />
```

Comment:
is this going to help? because it will run when the JS is executed by the browser, so like just a ms before the dom is rendered

Reply:
It should, as the page will render, the user still needs to interact with it. I will check it in the preview URL.